### PR TITLE
Routing free trial

### PR DESCRIPTION
**Pull request by super-ming**
I added my changes to Jesus's set up for the new home page in the routing-free-trial branch and combined it with Karl's recent code.

### CODE REVIEW

File: app/scripts/routes/FreeTrial.jsx [hunk 1]
```diff
@@ -6,14 +6,37 @@ export default class FreeTrial extends React.Component {
...
     };
     this.handleChange = this.handleChange.bind(this);
   }
 
   handleChange(event) {
+    console.log("event",event.target);
+    const {id, value}=event.target
     this.setState({
-      inputvalue: event.target.value,
+      [id]: value,
```
Comment (eragnew) [inline app/scripts/routes/FreeTrial.jsx]: Does this work? I've never seen it written like this, but also still pretty new to React.

File: app/scripts/routes/FreeTrial.jsx [hunk 2]
```diff
@@ -6,14 +6,37 @@ export default class FreeTrial extends React.Component {
...
+      },
+      allergies: '',
+      specialNeeds: ''
     };
     this.handleChange = this.handleChange.bind(this);
   }
 
   handleChange(event) {
+    console.log("event",event.target);
+    const {id, value}=event.target
```
Comment (eragnew) [inline app/scripts/routes/FreeTrial.jsx]: Should be `const {id, value} = event.target;` for consistency

File: app/scripts/routes/NewHome.jsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+import '../../styles/routes/_newhome.scss';
+
+export default class NewHome extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+      ticker: false,
+      opacity: false};
```
Comment (eragnew) [inline app/scripts/routes/NewHome.jsx]: Formatting should be:

```
this.state = {
  ticker: false,
  opacity: false
};
```

File: app/scripts/routes/FreeTrial.jsx (hunk 1 above)
Comment (super-ming) [reply app/scripts/routes/FreeTrial.jsx]: This is what Jesus and I came up with at the last meeting, and it seems to work.

**Pull request merged**